src/custom/pages/Account/Tokens/TokensOverview.tsx

```diff
@@ -21,10 +24,24 @@ export default function TokensOverview() {
     <Wrapper>
       <AccountMenu />
       <AccountPageWrapper>
-        <Subtitle>All tokens</Subtitle>
-        <Card>
+        <AccountHeading>
+          <Subtitle>Saved tokens</Subtitle>
+          <RemoveTokens onClick={() => removeAllSavedTokens()}>(Clear)</RemoveTokens>
+        </AccountHeading>
+        <AccountCard>
+          {savedTokens.length > 0 ? (
+            <TokensTable tokensData={savedTokens} />
+          ) : (
+            <MainText>Saved tokens will appear here</MainText>
```

Comment:
even if we don't do internationalisation yet, we should use the `Trans`  or the the `t

This way, when we do internationalisation, we will have less job to do

Reply:
Added this